### ISSUE TITLE
partner.ebay.com

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/CoreLibs#1204]
@adguard-bot commented on [Fri Nov 22 2019](https://github.com/AdguardTeam/AdguardFilters/issues/44547)

### Issue URL (Incorrect Blocking)
[https://partner.ebay.com/secure/mediapartner/campaigns/view-campaign-alerts-flow.ihtml?execution=e1s1](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpartner.ebay.com%2Fsecure%2Fmediapartner%2Fcampaigns%2Fview-campaign-alerts-flow.ihtml%3Fexecution%3De1s1)
### Comment
Username: @klepper
> Okay, so I am an eBay affiliate, and they just redesigned their site.  Adguard breaks it. Here is a video I made that may show it better: https://youtu.be/Gbd1uIolVz4
> 
> 

### Screenshots
<details>
  <summary>Screenshot 2</summary>

  ![Screenshot 2](https://reports-img.adguard.com/9ISIlSY.jpg)
</details>
<details>
  <summary>Screenshot 3</summary>

  ![Screenshot 3](https://reports-img.adguard.com/xpwjCpI.jpg)
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.3.4
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>List-KR
Other extensions used: | It's adguard, 100% sure. 

---

@klepper commented on [Fri Nov 22 2019](https://github.com/AdguardTeam/AdguardFilters/issues/44547#issuecomment-557554220)

can you remove screenshot 1? 

---

@AdamWr commented on [Fri Nov 22 2019](https://github.com/AdguardTeam/AdguardFilters/issues/44547#issuecomment-557571693)

Hi,
Sure, done.

It looks like a bug.
Could you please add this rule to user filter and check if it works:
`@@||partner.ebay.com^$elemhide,jsinject,extension`

By the way, I don't have account on ebay, so could you please tell if `partner.ebay.com` shows any ads?

---

@klepper commented on [Fri Nov 22 2019](https://github.com/AdguardTeam/AdguardFilters/issues/44547#issuecomment-557575831)

Boom, you fixed it!

![image](https://user-images.githubusercontent.com/9100213/69437947-7a86bb80-0d01-11ea-9b22-11f572285ff4.png)

> By the way, I don't have account on ebay, so could you please tell if `partner.ebay.com` shows any ads?

No, but you know, when a website doesn't work right the first step is to disable adguard to troubleshoot. 

Thanks! 

---

@AdamWr commented on [Fri Nov 22 2019](https://github.com/AdguardTeam/AdguardFilters/issues/44547#issuecomment-557586393)

Ok, I'm glad that it works.
I will add this rule to AdGuard Base filter, update should be available in 1-2 hours.

Thank you for your reply.

**Comment by AdamWr** [comment]
It seems that account is not required, at least for issue from screenshot 2/3.
Steps to reproduce:
1. Disable this rule (from Base filter):
`@@||partner.ebay.com^$elemhide,jsinject,extension`
2. Go here - https://partner.ebay.com/login.user

There should be sign in button (like on screenshot 2), but it doesn't appear.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/69440427-20114d80-0d49-11ea-89ba-381b5741bb95.png)

</details>

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/CoreLibs/issues/1221
